internal/webhook/podmutationhandler_suite_test.go

```diff
@@ -425,7 +425,7 @@ func TestPodMutationHandler_Handle(t *testing.T) {
 							Env: []corev1.EnvVar{
 								{Name: "a", Value: "a"},
 								{Name: "b", Value: "b"},
-								{Name: "PHP_INI_SCAN_DIR", Value: "/newrelic-instrumentation/php-agent/ini"},
+								{Name: "PHP_INI_SCAN_DIR", Value: ":/newrelic-instrumentation/php-agent/ini"},
```

Comment:
This is not correct- the default value in php.go is `/newrelic-instrumentation/php-agent/ini` without the leading `:` character. Testing with this value risks missing a mis-injected `PHP_INI_SCAN_DIR` value.